### PR TITLE
Table component

### PR DESCRIPTION
**Pull request by IlanSirisky**

### CODE REVIEW

File: src/components/Table/Table.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import { Table as MuiTable, TableContainer, Paper } from "@mui/material";
+import TableHead from "./TableHead";
+import TableBody from "./TableBody";
+import { TableWrapper } from "./styles";
+import { IColumnLabels } from "./types";
+import CustomTablePagination from "./TablePagination";
+
+interface TableProps {
+  columnTitles: IColumnLabels[];
+  data: any[];
```
Comment (oranroz11) [inline src/components/Table/Table.tsx:13]: please don't use any - create a type

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  rowPerPageOptions?: number[];
+  sx?: object;
```
Comment (oranroz11) [inline src/components/Table/Table.tsx:15]: please create a type

File: src/components/Table/TableBody.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { TableBody as MuiTableBody, TableCell, TableRow } from "@mui/material";
+import { DataCellStyle, DataCellWrapper } from "./styles";
+import { BodyRegular } from "../../styles/typography";
+import { IColumnLabels } from "./types";
+
+interface TableBodyProps {
+  data: any[];
```
Comment (oranroz11) [inline src/components/Table/TableBody.tsx:7]: add type

File: src/components/Table/styles.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  lineHeight: "22px",
+  fontSize: "14px",
+  gap: `${cssSpacings.s8}`,
+};
+
+export const DataCellStyle = {
+  paddingLeft: "0",
+  paddingTop: "8px",
+  paddingBottom: "8px",
+  paddingRight: "20px",
```
Comment (oranroz11) [inline src/components/Table/styles.ts]: use padding attribute that includes all padding part and use cssSpacings

File: src/components/Table/styles.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import { cssFontWeights, cssSpacings } from "../../styles/stylingValues";
+import { COLORS } from "../../styles/colors";
+import { flexRow } from "../../styles/globalStyles";
+
+export const TableWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-bottom: 38px;
```
Comment (oranroz11) [inline src/components/Table/styles.ts]: use cssSpacings

File: src/constants/table.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import { IColumnLabels } from "../components/Table/types";
+import PokemonAvatarTableCell from "../features/listView/TableCell/PokemonAvatarTableCell";
+import PokemonIdTableCell from "../features/listView/TableCell/PokemonIdTableCell";
+import { IPokemonData } from "../types/pokemonTypes";
+
+export const pokemonTableColumnLabels: IColumnLabels[] = [
+  {
+    value: "title",
+    label: "Pokemon name",
+    padding: "0 0 0 70px",
```
Comment (oranroz11) [inline src/constants/table.tsx]: use cssSpacings

File: src/constants/table.tsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    width: "30%",
+    component: (row: IPokemonData) => <PokemonAvatarTableCell row={row} />,
+  },
+  {
+    value: "id",
+    label: "ID",
+    padding: "0 40px 0 0",
```
Comment (oranroz11) [inline src/constants/table.tsx]: use cssSpacings

**Review by oranroz11 — CHANGES_REQUESTED** [review]

File: src/components/Table/Table.tsx (hunk 1 above)
Comment (IlanSirisky) [reply src/components/Table/Table.tsx:13]: Yuval is aware, said to leave it like that

File: src/components/Table/Table.tsx (hunk 2 above)
Comment (IlanSirisky) [reply src/components/Table/Table.tsx:15]: Yuval is aware, said to leave it like that

File: src/components/Table/TableBody.tsx (hunk 3 above)
Comment (IlanSirisky) [reply src/components/Table/TableBody.tsx:7]: Yuval is aware, said to leave it like that

**Review by YuvalLeviMoveoHLS — APPROVED** [review]

**Pull request merged**